src/plugins/intel_cpu/src/emitters/plugin/x64/jit_load_store_emitters.cpp

```diff
@@ -372,11 +372,15 @@ void jit_load_emitter::load_bytes(const Vmm& vmm, const Xbyak::Reg64& reg, int o
         }
 
         if (has_xmm_block) {
-            h->vinsertf128(ymm, ymm, xmm, 1);  // insert to upper bits of ymm
+            // insert to upper bits of ymm
+            // on avx512 target, ymm may have index [16,31], so need evex encoded instruction.
+            mayiuse(cpu::x64::avx512_core) ? h->vinsertf32x4(ymm, ymm, xmm, 1) : h->vinsertf128(ymm, ymm, xmm, 1);
```

Comment:
Minor: to improve readability:
```suggestion
            if (mayiuse(cpu::x64::avx512_core))
                  h->vinsertf32x4(ymm, ymm, xmm, 1) ;
            else
                  h->vinsertf128(ymm, ymm, xmm, 1);
```

The same suggestion for line below. I understand that current solutions uses only one line but suggested lines are aligned with the code in this file. But I don't insist 😊

Reply:
Code style is aligned, thanks Alexandra!